### DISCUSSION TITLE
Virtual Env is not respected

### DISCUSSION THREAD

**Discussion by jomach** [tiangolo/uvicorn-gunicorn-fastapi-docker#308] [Questions] [answered]
Hey, 

I have a dockerfile like this: 
...
FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8 AS service
COPY --from=test_runner /venv /venv
ENV PATH=/venv/bin:$PATH

When I run the container It cannot find my installed packaged.

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image

That would probably make it easier to debug things like this. :nerd_face:
